Makefile

```diff
@@ -244,19 +251,23 @@ WEBSITE_ASSETS_EXAMPLES = \
   tutorial_en/tutorial_en.html \
   tutoriel_fr/tutoriel_fr.html \
   us_tax_code/us_tax_code.html \
-  allocations_familiales/allocations_familiales.html \
-  allocations_familiales/allocations_familiales_schema.json \
-  aides_logement/aides_logement.html \
-  aides_logement/aides_logement_schema.json
+  allocations_familiales/Allocations_familiales.html \
+  allocations_familiales/Allocations_familiales_schema.json \
+  aides_logement/Aides_logement.html \
+  aides_logement/Aides_logement_schema.json
 
 WEBSITE_ASSETS_ALL = $(WEBSITE_ASSETS) $(addprefix catala-examples.tmp/,$(WEBSITE_ASSETS_EXAMPLES))
 
 website-assets-base: build
 	$(call local_tmp_clone,catala-examples) && \
-	dune build $(addprefix _build/default/,$(WEBSITE_ASSETS_ALL)) --profile=release
-
-website-assets.tar:
-	# $(MAKE) DUNE_PROFILE=release website-assets-base
+	$(MAKE) -C catala-examples.tmp \
```

Comment:
Ah yes it could be added, but the rule will anyway remove it right after termination (because of https://github.com/CatalaLang/catala/pull/586/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R297), unless you kill make very violently.